prime-router/src/main/kotlin/azure/SubmissionsFacade.kt

```diff
@@ -0,0 +1,104 @@
+package gov.cdc.prime.router.azure
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import gov.cdc.prime.router.ActionResponse
+import gov.cdc.prime.router.Metadata
+import gov.cdc.prime.router.Submission
+import gov.cdc.prime.router.SubmissionsProvider
+import java.time.OffsetDateTime
+
+/**
+ * Submissions / history API
+ * Contains all business logic regarding submissions and JSON serialization.
+ */
+
+// TODO: Add test/SubmissionsFacadeTests
+class SubmissionsFacade(
+    private val db: DatabaseSubmissionsAccess = DatabaseSubmissionsAccess()
+) : SubmissionsProvider {
+
+    // Ignoring unknown properties because we don't require them. -DK
+    private val mapper = jacksonObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+
+    init {
+        // Format OffsetDateTime as an ISO string
+        mapper.registerModule(JavaTimeModule())
+        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+    }
+
+    override fun findSubmissionsAsJson(
+        organizationName: String,
+        limit: Int
+    ): String {
+        val result = findSubmissions(organizationName, limit)
```

Comment:
split this out, let the parent (the Azure Function) compose the pieces.

So it will be able to call `findSubmissions` then serialize them.

Reply:
This is also following patterns elsewhere in the codebase (mainly the Settings Facade) and I imagine will be a useful structure when we add parameter filters and the various methods of pagination.

Reply:
separating it allows the code that care's about the type of the response (the Azure Function) to define it.